### ISSUE TITLE
[Mumble] Admin panels are incompatible with php7

### ISSUE THREAD

**Issue by qq7** [turnkeylinux/tracker#1052]
I'm unable to find any Mumble admin panel which would work with php7 and stretch [php-zeroc-ice](https://packages.debian.org/stretch/php-zeroc-ice) package. 

@MPTMG do you have any suggestions on this? As for now it seems that the best decision is to drop admin panels from the appliance at all.

**Comment by JedMeister** [comment]
Thanks @qq7. I've had a quick google myself and I can't see anything either. :cry: It's a bit of a pity, but I guess that so long as it provides some commandline tools for setting things up, then it's still worth something... A web interface would certainly be better though!

**Comment by MPTMG** [comment]
Mumble-Django but last release was with Debian 7. I could look into adding old repo and integrating it instead of php based interfaces.

**Comment by MPTMG** [comment]
Actually it was available in jessie as well. https://packages.debian.org/jessie/mumble-django

Not using the package.... it could always be installed from source as well if preferable. Python based so I doubt one would run into any compatibility issues.

It should be compatible with stretch there just isn't a release as far as I know.

**Comment by MPTMG** [comment]
There is this option, but technically speaking it is not a traditional web admin application. https://github.com/alfg/guildbit
Let me know if there is interest in integrating this and I will make it happen.

**Comment by MPTMG** [comment]
I thought I already posted this, but I don't see it, so perhaps I forgot to hit comment. Tried Mumble-django on stretch and unfortunately it requires python-django<1.8. Stretch has an incompatible/newer version of python-django and it would require more effort than it is worth to get it working. My vote is to drop web interface support until compatible web interface is available. I will do the work once a compatible interface is available.
